st2common/tests/fixtures/print_to_stdout_stderr_sleep.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright 2021 The StackStorm Authors.
+# Copyright 2019 Extreme Networks, Inc.
```

Comment:
Just a comment on copyright. If this is a new file then surely it should only have copyright from the date it was created?

Reply:
I actually just followed example from existing files (they didn't seem to update Extreme line), but that's a good / interesting question.

Technically since it's a new file, Extreme doesn't hold a copyright to it so it could probably be removed (but would be good to double check with LF legal and keeping track of that may be a bit hard).

Also, a lot of projects yearly update copyright strings in all the files so they reflect start - current year (e.g. 2014-2021).

Reply:
That's right. I can confirm that for all the new files we just use the StackStorm Authors and don't need Extreme.

```
# Copyright 2021 The StackStorm Authors.
```